test_graphyte.py

```diff
@@ -141,6 +142,31 @@ def test_timestamp_generated(self):
         timestamp = int(match.group(1))
         self.assertTrue(send_time - 2 <= timestamp <= send_time + 2)
 
+    def test_send_socket_do_raise_error(self):
+        class SenderWithError(graphyte.Sender):
+            def send_message(self, message):
+                raise RuntimeError()
+
+        sender = SenderWithError('dummy_host', raise_send_errors=True)
+        with self.assertRaises(RuntimeError):
+            sender.send_socket('test')
+
+    def test_send_socket_do_not_raise_error(self):
+        class SenderWithError(graphyte.Sender):
+            def send_message(self, message):
+                raise RuntimeError()
+
+        sender = SenderWithError('dummy_host')
+        try:
+            sender.send_socket('test')
+        except RuntimeError:
+            self.fail('send_socket() raised an exception')
+
+    def test_sender_disable_raise_in_async(self):
+        sender = TestSender(interval=1)
+        self.assertFalse(sender.raise_send_errors)
```

Comment:
I'm not sure what this is testing. Should this test be changed to a test that checks that it raises `ValueError` if both interval and raise_send_errors are provided? (In which case you'll need to use `graphyte.Sender` and not `TestSender`.

Reply:
My bad.  I forgot to change that test.  It was testing the logic I replaced with the value error.